### PR TITLE
Create new version manifest for KDEConnect 1.4-449

### PR DESCRIPTION
**Pull request by pbrandstetter**
Currently most recent version for KDEConnect-installer

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/5760)

### CODE REVIEW

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

File: manifests/KDE/KDEConnect/1.4-449.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Id: KDE.KDEConnect
+Version: 1.4-449
+Name: KDE Connect
+Publisher: KDE
+License: GPL-2.0
+LicenseUrl: https://github.com/KDE/kdeconnect-kde/blob/master/COPYING
```
Comment (ItzLevvie) [inline manifests/KDE/KDEConnect/1.4-449.yaml:6]:
```suggestion
LicenseUrl: https://github.com/KDE/kdeconnect-kde/blob/master/LICENSES/GPL-2.0-or-later.txt
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by sa-orlov — APPROVED** [review]

**Review by sa-orlov — APPROVED** [review]

File: manifests/KDE/KDEConnect/1.4-449.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Name: KDE Connect
+Publisher: KDE
+License: GPL-2.0
+LicenseUrl: https://github.com/KDE/kdeconnect-kde/blob/master/LICENSES/GPL-2.0-or-later.txt
+Description: Enabling communication between all your devices. Made for people like you.
+Homepage: https://kdeconnect.kde.org/
+Tags: connect,phone,mobile,android,notification
+Installers:
+  - Arch: x64
+    Url: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/lastSuccessfulBuild/artifact/kdeconnect-kde-1.4-449-windows-msvc2019_64-cl-sideload.appx
```
Comment (ItzLevvie) [inline manifests/KDE/KDEConnect/1.4-449.yaml:12]:
```suggestion
    Url: https://binary-factory.kde.org/job/kdeconnect-kde_Release_win64/449/artifact/kdeconnect-kde-1.4-449-windows-msvc2019_64-cl-sideload.appx
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/KDE/KDEConnect/1.4-449.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/KDE/KDEConnect/1.4-449.yaml:12]: a new installer (version 1.4-450) was released an hour ago, so using `lastSuccessfulBuild` will always pick up the latest installer  every day regardless of the version listed in the manifest.

Comment (pbrandstetter) [reply manifests/KDE/KDEConnect/1.4-449.yaml]: Ok, thanks. Fixed it and re-validated the manifest!

**Pull request merged**